src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -200,13 +207,13 @@ public boolean isEmpty()
 
     /**
      * Makes an iterator for the list.
+     *
      * @return an iterator for the colonies.
      */
     @Override
     public Iterator<T> iterator()
     {
-        return new Iterator<T>()
-        {
+        return new Iterator<T>() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)